unitTests/controllers/MLFacetQueryControllerTest.ts

```diff
@@ -131,5 +131,21 @@ export function MLFacetQueryControllerTest() {
 
       expect(latestFacetRequest().numberOfValues).toBe(latestFacetRequest().currentValues.length);
     });
+
+    it('isFieldExpanded should be false by default', () => {
+      expect(latestFacetRequest().isFieldExpanded).toBe(false);
+    });
+
+    it(`when more values are requested than the numberOfValues options
+      isFieldExpanded should be false`, () => {
```

Comment:
The test description should say `isFieldExpanded should be true` in this case.

Reply:
that copy paste :(
Thanks!